php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S5783.html

```diff
@@ -0,0 +1,27 @@
+<p>When verifying that code raises an exception, a good practice is to avoid having multiple method calls inside the tested code, to be explicit about
+what is exactly tested.</p>
+<p>When two of the methods can raise the same <strong>checked</strong> exception, not respecting this good practice is a bug, since it is not possible
```

Comment:
In PHP there is no such thing as "checked" and "unchecked". Maybe remove "checked". (Also in the title)